lib/ext/fhir/library.rb

```diff
@@ -0,0 +1,23 @@
+# Custom functionalty for fhir-mongoid-models/app/models/fhir/library.rb
+module FHIR
+  class Library < DomainResource
+
+    def create_data_elements(fhir_value_sets)
+      dataRequirement.map do |dataReq|
+        type = dataReq.type.value
+        oid = dataReq.codeFilter.first&.valueSet&.value&.match(/([0-2])((\.0)|(\.[1-9][0-9]*))*$/).to_s
```

Comment:
Codacy found an issue: [Use snake_case for variable names.](https://app.codacy.com/gh/MeasureAuthoringTool/fhir-mongoid-models/pullRequest?prid=5999190)